tests/conftest.py

```diff
@@ -58,7 +58,7 @@ def create_contract(path, args=(), sender=t.k0):
         bytecode = u.decode_hex(hexcode)
         ct = ContractTranslator(abi)
         code = bytecode + (ct.encode_constructor_arguments(args) if args else b'')
-        address = t.chain.tx(sender=sender, to=b'', startgas=(4 * 10 ** 6), value=0, data=code)
+        address = t.chain.tx(sender=sender, to=b'', startgas=(4 * 10 ** 6 + 5 * 10 ** 5), value=0, data=code)
```

Comment:
Why do we need that increase? It didn't work without it?

Reply:
yes, this change lead to increased cost of deployment of the contract